site/sst-env.d.ts

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
```

Comment:
is this needed?

Reply:
Removed this file as @lj-kpmg  will be adding this file